index.bs

```diff
@@ -2300,6 +2302,85 @@ slots</a> described in the following table:
         1. Return <code>wMaxPacketSize</code> of |endpointDescriptor|.
 </dl>
 
+# The USB Blocklist # {#blocklist}
+
+<xmp class="idl">
+  // USBBlocklistEntry is never exposed.
+  dictionary USBBlocklistEntry {
+    required unsigned short idVendor;
+    required unsigned short idProduct;
+    required unsigned short bcdDevice;
+  };
+</xmp>
+
+This specification relies on a <a href="https://raw.githubusercontent.com/WICG/webusb/main/blocklist.txt">blocklist.txt</a>
+file in this repository to restrict the set of devices a website can access.
+
+The result of <dfn>parsing the blocklist</dfn> at a {{URL}} |url| is a
+{{Promise}} that resolves to a [=list=] of {{USBBlocklistEntry}} objects,
```

Comment:
I don't think we should use the `Promise` machinery here. The other specifications that provide steps for parsing a blocklist don't do this. I'm guessing you wanted to be able to link to the [fetch](https://fetch.spec.whatwg.org/#concept-fetch) algorithm but I don't think that algorithm is appropriate because we're not fetching the resource on behalf of a document.

The use of a `Promise` here also makes the definition of "USB blocklist" a little ambiguous because it somewhat implies that the steps referring to the blocklist could block on fetching the list, which really isn't the intent here.

Reply:
Removed `Promise` and fetch, please take another look